rma/models/rma_order_line.py

```diff
@@ -98,12 +101,24 @@ def _get_rma_move_qty(self, states, direction="in"):
             product_obj = self.env["uom.uom"]
             qty = 0.0
             if direction == "in":
-                op = ops["="]
-            else:
-                op = ops["!="]
-            for move in rec.move_ids.filtered(
-                lambda m: m.state in states and op(m.location_id.usage, rec.type)
-            ):
+                moves = rec.move_ids.filtered(
+                    lambda m: m.state in states
+                    and (
+                        m.location_id.usage == "supplier"
+                        or m.location_id.usage == "customer"
+                    )
+                    and m.location_dest_id.usage == "internal"
```

Comment:
```suggestion
                    and (m.location_dest_id.usage == "internal"
                        or m.location_dest_id_usage == "supplier"
                    )
```
When the RMA is customer to supplier it is not getting the moves correctly, because the location_id.usage is "customer" but the location_dest_id.usage is "supplier". and does take into account this option.

Reply:
Also change the following function, it seems that the field "supplier_rma_line_ids" never gets filled.
```
@api.model
     def _get_supplier_rma_qty(self):
-        return sum(
-            self.supplier_rma_line_ids.filtered(lambda r: r.state != "cancel").mapped(
-                "product_qty"
-            )
-        )
+        return self._get_rma_move_qty("done", direction="in")
```